src/testController.ts

```diff
@@ -22,6 +22,11 @@ export class TestController {
   private terminal: vscode.Terminal | undefined;
   private ruby: Ruby;
   private telemetry: Telemetry;
+  // We allow the timeout to be configured in seconds, but exec expects it in milliseconds
+  private testTimeout =
```

Comment:
In order to add the time in the message, we need it in seconds. So, I made `testTimeout` be in seconds so that we can multiply it by 1000 when we run `exec`.

I think it'd be weird dividing it by 1000 just for the message.